site2/docs/io-elasticsearch-sink.md

```diff
@@ -16,11 +48,45 @@ The configuration of the ElasticSearch sink connector has the following properti
 |------|----------|----------|---------|-------------|
 | `elasticSearchUrl` | String| true |" " (empty string)| The URL of elastic search cluster to which the connector connects. |
 | `indexName` | String| true |" " (empty string)| The index name to which the connector writes messages. |
+| `schemaEnable` | Boolean | false | false | Turn on the Schema Aware mode. |
+| `createIndexIfNeeded` | Boolean | false | false | Manage index if missing. |
+| `maxRetries` | Integer | false | 1 | The maximum number of retries for elasticsearch requests. Use -1 to disable it.  |
+| `retryBackoffInMs` | Integer | false | 100 | The base time in milliseconds to wait when retrying an elasticsearch request. |
+| `maxRetryTimeInSec` | Integer| false | 86400 | The maximum retry time interval in seconds for retrying an elasticsearch request.. |
+| `bulkEnabled` | Boolean | false | false | Enable the elasticsearch bulk processor to flush write requests based on the number or size of requests, or after a given period. |
+| `bulkActions` | Integer | false | 1000 | The maximum number of actions per elasticsearch bulk request. Use -1 to disable it. |
+| `bulkSizeInMb` | Integer | false |5 | The maximum size in megabytes of elasticsearch bulk requests.Use -1 to disable it. |
+| `bulkConcurrentRequests` | Integer | false | 0 | The maximum number of in flight elasticsearch bulk requests. The default 0 allows the execution of a single request. A value of 1 means 1 concurrent request is allowed to be executed while accumulating new bulk requests. |
+| `bulkFlushIntervalInMs` | Integer | false | -1 | The bulk flush interval flushing any bulk request pending if the interval passes. Default is -1 meaning not set. |
```

Comment:
what's the meaning of `The bulk flush interval flushing any bulk request pending if the interval passes.`? can you rewrite this sentence?

Reply:
I have reworded the sentence PTAL